src/Components/ExpandedRulesDetails.js/ExpandedRulesDetails.js

```diff
@@ -20,11 +20,20 @@ import PropTypes from 'prop-types';
 import TemplateProcessor from '@redhat-cloud-services/frontend-components-advisor-components/TemplateProcessor/TemplateProcessor';
 import ObjectsModal from '../ObjectsModal/ObjectsModal';
 import { ObjectsTableColumns } from '../../AppConstants';
+import _ from 'lodash';
```

Comment:
```suggestion
import isEmpty from 'lodash/isEmpty';
```

This helps to load only what is needed.